tests/framework/utils/k8s_helper.go

```diff
@@ -1171,7 +1171,7 @@ func (k8sh *K8sHelper) WaitUntilPVCIsDeleted(namespace string, pvcname string) b
 	ctx := context.TODO()
 	for i := 0; i < RetryLoop; i++ {
 		_, err := k8sh.Clientset.CoreV1().PersistentVolumeClaims(namespace).Get(ctx, pvcname, getOpts)
-		if err != nil && kerrors.IsNotFound(err) {
+		if kerrors.IsNotFound(err) {
```

Comment:
I think the `err != nil` part is important to make sure this doesn't segfault. Why was this change made?

Reply:
done reverted it back.
I don't remember why this was changed.